diskfs_unix.go

```diff
@@ -13,14 +13,14 @@ import (
 func getSectorSizes(f *os.File) (int64, int64, error) {
 	/*
 		ioctl(fd, BLKPBSZGET, &physicalsectsize);
-
 	*/
 	fd := f.Fd()
-	logicalSectorSize, err := unix.IoctlGetInt(int(fd), blksszGet)
+
```

Comment:
Looks like you added an extra line here.

Reply:
```suggestion
- 
```